pkg/ebpf/offsetguess.go

```diff
@@ -93,41 +96,12 @@ type fieldValues struct {
 	daddrIPv6 [4]uint32
 }
 
-func startServer() (chan struct{}, uint16, error) {
-	// port 0 means we let the kernel choose a free port
-	addr := fmt.Sprintf("%s:0", listenIP)
-	l, err := net.Listen("tcp4", addr)
-	if err != nil {
-		return nil, 0, err
-	}
-	lport, err := strconv.Atoi(strings.Split(l.Addr().String(), ":")[1])
-	if err != nil {
-		return nil, 0, err
-	}
-
-	stop := make(chan struct{})
-	go acceptV4(l, stop)
-
-	return stop, uint16(lport), nil
-}
-
-func acceptV4(l net.Listener, stop chan struct{}) {
-	for {
-		_, ok := <-stop
-		if ok {
-			conn, err := l.Accept()
-			if err != nil {
-				l.Close()
-				return
-			}
-			conn.Close()
-		} else {
-			// the main thread closed the channel, which signals there
-			// won't be any more connections
-			l.Close()
-			return
-		}
+func offsetGuessProbes(c *Config) []KProbeName {
```

Comment:
Any reason not to keep this with the other config code?

Reply:
not really. I just thought this made more sense here, but I'm happy to turn this into something like  `Config.OffsetGuessProbes()` if you think it makes more sense.